cosmos-cmd-tlm-api/app/controllers/narrative_controller.rb

```diff
@@ -79,7 +79,7 @@ def search
       start, stop = parse_time_input(x_start: params[:start], x_stop: params[:stop])
       q = params[:q]
       raise NarrativeInputError "Must include q value" if q.nil?
```

Comment:
Still Narrative

Reply:
Yeah I haven't made it up the chain yet to the controller and frontend